ztest/ztest.go

```diff
@@ -135,11 +136,28 @@ func Run(t *testing.T, dirname string) {
 // ZTest defines a ztest.
 type ZTest struct {
 	ZQL          string `yaml:"zql"`
-	Input        string `yaml:"input"`
+	Input        Inputs `yaml:"input"`
 	OutputFormat string `yaml:"output-format,omitempty"`
 	Output       string `yaml:"output"`
 }
 
+type Inputs []string
+
+func (i *Inputs) UnmarshalYAML(value *yaml.Node) error {
+	if value.Kind == yaml.SequenceNode {
+		var inputs []string
+		err := value.Decode(&inputs)
+		*i = inputs
+		return err
+	}
+	var input string
+	if err := value.Decode(&input); err != nil {
+		return err
+	}
+	*i = append(*i, input)
+	return nil
```

Comment:
If `value.Kind` is a `yaml.SequenceNode`, the code sets `i` regardless of the value of `err` from `value.Decode()`. But if `value.Kind` is something else, `i` is only modified if `err` is `nil`. Any particular reason for this difference?

Reply:
I would imagine that `value.Decode` would throw an error if the underlying yaml isn't compatible with the provided interface (i.e. "string").